README.md

```diff
@@ -104,6 +104,66 @@ const [templateFile, setTemplateFile] = useState<File | null>(null);
   setTemplateFile(e!.target!.files!.item(0));
 }}/>
 ```
+### API
+`connected-workbook` exposes __WorkbookManager__, which generates a workbook with a single loaded query or table data.
```

Comment:
The library exposes a __workbookManager__, which generates a workbook via several APIs: